boards/STM32F407ZET6-STM32_F4XX.md

```diff
@@ -0,0 +1,452 @@
+| 1     | +3.3V rail   | VCC   | -     |
+| 2     | +3.3V rail   | VCC   | -     |
+| 3     | PB4          | TRST  | -     |
+| 4     | Ground plane | GND   | -     |
+| 5     | PA15         | TDI   | -     |
+| 6     | Ground plane | GND   | -     |
+| 7     | PA13         | TMS   | SWDIO |
+| 8     | Ground plane | GND   | -     |
+| 9     | PA14         | TCLK  | SWCLK |
+| 10    | Ground plane | GND   | -     |
+| 11    | N.C.         | RTCK  | -     |
+| 12    | Ground plane | GND   | -     |
+| 13    | PB3          | TDO   | SWO   |
+| 14    | Ground plane | GND   | -     |
+| 15    | NRST         | RESET | -     |
+| 16    | Ground plane | GND   | -     |
+| 17    | N.C.         | N.C.  | -     |
+| 18    | Ground plane | GND   | -     |
+| 19    | N.C.         | N.C.  | -     |
+| 20    | Ground plane | GND   | -     |
+
+## TFT
+
+| Pin   | Connected to | Function       |
+| ----- | ------------ | -------------- |
+|   2   | NRST         |                |
+|   1   | Ground plane |                |
+|   4   | PD9          | FSMC D14       |
+|   3   | PD10         | FSMC D15       |
+|   6   | PE15         | FSMC D12       |
+|   5   | PD8          | FSMC D13       |
+|   8   | PE13         | FSMC D10       |
+|   7   | PE14         | FSMC D11       |
+|  10   | PE11         | FSMC D8        |
+|   9   | PE12         | FSMC D9        |
+|  12   | PE9          | FSMC D6        |
+|  11   | PE10         | FSMC D7        |
+|  14   | PE7          | FSMC D4        |
+|  13   | PE8          | FSMC D5        |
+|  16   | PD0          | FSMC D2        |
+|  15   | PD1          | FSMC D3        |
+|  18   | PD14         | FSMC D0        |
+|  17   | PD15         | FSMC D1        |
+|  20   | PD5          | FSMC NWE       |
+|  19   | PD4          | FSMC NOE       |
+|  22   | PG12         | FSMC NE4       |
+|  21   | PF12         | FSMC A6        |
+|  24   | PC13         | Touch CS       |
+|  23   | PB0          | Touch CLK      |
+|  26   | PB2/BOOT1    | Touch (MI)SO   |
+|  25   | PF11         | Touch (MO)SI   |
+|  28   | PB15 via Q2  | Backlight      |
+|  27   | PB1 via R17  | Touch Event    |
+|  30   | Ground plane |                |
+|  29   | nc           |                |
+|  32   | Ground plane |                |
+|  31   | +3.3V rail   |                |
+
+## NRF2401
+
+| Pin   | Connected to |
+| ----- | ------------ |
+| GND   | Ground plane |
+| VCC   | +3.3V rail   |
+| CE    | PG6          |
+| CSN   | PG7          |
+| SCK   | PB3          |
+| MOSI  | PB5          |
+| MISO  | PB4          |
+| IRQ   | PG8          |
+
+## SD-card
+
+| Pin |          | Connected to |
+| --- | -------- | ------------ |
+|  1  | SDIO_D2  | PC10         |
+|  2  | SDIO_D3  | PC11         |
+|  3  | SDIO_CMD | PD2          |
+|  4  | VCC      | +3.3V rail   |
+|  5  | SDIO_SCK | PC12         |
+|  6  | GND      | Ground plane |
+|  7  | SDIO_D0  | PC8          |
+|  8  | SDIO_D1  | PC9          |
+|  9  | nc       | Ground plane |
```

Comment:
N.C.

Reply:
The info is from the schematic diagram and somewhat confusing. After looking at it for a LONG time I think  they are trying to say that there is no Card detect switch in the socket they are using, therefore N.C.. The pin on the socket is actually connected to Ground Plane. As it is now I removed the 'nc' and left that field open, but perhaps it would be better to say '(Card Detect)'  with  'Ground Plane' in the 'connected to' field ?

Reply:
It would indeed be better to mark this as Card Detect since it is a standard thing in most sockets.

I also experienced that it is hard to find solid information about SD-card sockets in general. The pinout of SD cards can be found anywhere, but the pinout of sockets is much harder to find.